src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -1414,6 +1427,16 @@ export async function update(
     updateOnly: true,
     siweValues: args?.siweValues
   })
+
+  const isMod = gameInfo.type === 'mod'
```

Comment:
this should be if game is market wars for now since it is not general to all mods